cric-predict/src/Components/Leagues.js

```diff
@@ -1,34 +1,57 @@
 import React from 'react';
 import '../Stylesheets/App.css'
 
-function Leagues() {
+export default class League extends React.Component  {
 
-  const matches = ['League 1', 'League 2', 'League 3', 'League 4'];
+  constructor(props){
+    super(props);
+    this.state = {leagues: []};
+  }
 
-  const list = []
+  componentDidMount() {
+    const apiUrl = 'http://127.0.0.1:8000/api/contest/league/';
+    fetch(apiUrl, {mode: 'no-cors'})
```

Comment:
is this API call being used? if yes then why do we have `mode: 'no-cors'`? and why is commented out try catch here?